pytensor/link/pytorch/linker.py

```diff
@@ -77,11 +90,11 @@ def __del__(self):
         self.gen_functors = []
 
         # Torch does not accept numpy inputs and may return GPU objects
-        def fn(*inputs, inner_fn=inner_fn):
+        def create_outputs(*inputs, inner_fn=inner_fn):
```

Comment:
Why the new name? Seems less clear

Reply:
Yeah this `fn` was shadowing a local variable `fn` so i just renamed one of them.

Reply:
Sure but can we use a different name. This doesn't "create_outputs" it converts inputs to torch tensors and outputs back to pytensor-compatible types

Reply:
sure thing - I can also just keep the shadowing lol. It's not the end of the world. 

From your description I would probably have called it `convert_types`  or smth.

Reply:
We can also put it inside the wrapper __call__ I guess?

Reply:
Sure, that makes sense too.